technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,72 @@
+# Continuous Profiling in TEAMMATES
+
+Authors: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Amrut Prabhu](https://github.com/amrut-prabhu) and [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [Overview of the Proposed Solution](#Overview-of-Solution)
+* [Tools considered for Performance Testing](#Tools-considered-for-Performance-Testing)
+* [Reasons for using JMeter](#Reasons-for-using-JMeter)
+* [Current implementation of the solution](#current-implementation-of-the-solution)
+* [Findings and Recommendations](#findings-and-recommendations)
+* [Future Work](#Future-work)
+
+## Introduction
+
+This report gives a brief overview of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes the reasons behind our proposed solution.
+
+## Problem
+
+TEAMMATES is one of the biggest student projects in the open source community. Currently, TEAMMATES boasts of a community comprising over 450 developers and a codebase 
+of nearly 130LoC. Maintaining such a project demands high quality standards to ensure long term survival. This means, 
+continuously monitoring code health and product performance. As the number of developers and user base continue to grow,
+we need to ensure optimal performance at all times. In this report, we propose a viable solution to perform regression 
+tests that will help developers keep a track of the potential bottlenecks and areas of optimizations. This will help
+boost the performance as the product evolves over time.
+
+## Overview of Solution
+
+The idea behind L&P tests is to simplify the process of understanding production performance and enable the developers to address bottlenecks before they become genuine production issues.
+Implementing these tests involves a few key points:-
+
+* A tool/software to help perform these tests
+* A database to store the data of the profiler
```

Comment:
One or both of these terms are not accurately used in this sentence: `database`, `profiler`.